ddev/ddev-browsersync
# Improving WordPress support

## The Issue
https://github.com/ddev/ddev-browsersync/issues/51

## How This PR Solves The Issue
This PR aims to replicate a similar approach used by ddev-redis to modify the project's CMS config.

## Manual Testing Instructions
1. `ddev start` a test project
2. Install the add on fork `ddev get ~/folder-of-fork/ddev-browsersync`
3. Start browsersync with `ddev browsersync` 
4. Open browsersync in your browser `ddev launch :3000`

WordPress should recognise the URL and load the page at port 3000 correctly.

## Related Issue Link(s)
https://github.com/ddev/ddev-browsersync/issues/51
